packages/google_mlkit_document_scanner/android/src/main/java/com/google_mlkit_document_scanner/GoogleMlKitDocumentScannerPlugin.java

```diff
@@ -1,55 +1,19 @@
 package com.google_mlkit_document_scanner;
 
-import android.app.Activity;
-import android.content.Intent;
-import android.content.IntentSender;
-import android.net.Uri;
-
 import androidx.annotation.NonNull;
-import androidx.annotation.Nullable;
-
-import com.google.android.gms.tasks.OnFailureListener;
-import com.google.android.gms.tasks.OnSuccessListener;
-import com.google.mlkit.vision.documentscanner.GmsDocumentScanner;
-import com.google.mlkit.vision.documentscanner.GmsDocumentScanning;
-import com.google.mlkit.vision.documentscanner.GmsDocumentScannerOptions;
-import com.google.mlkit.vision.documentscanner.GmsDocumentScanningResult;
-
-import java.net.URI;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
-import java.util.ArrayList;
 
-import io.flutter.Log;
 import io.flutter.embedding.engine.plugins.activity.ActivityAware;
 import io.flutter.embedding.engine.plugins.activity.ActivityPluginBinding;
 import io.flutter.embedding.engine.plugins.FlutterPlugin;
-import io.flutter.plugin.common.MethodCall;
 import io.flutter.plugin.common.MethodChannel;
-import io.flutter.plugin.common.PluginRegistry;
-
-public class GoogleMlKitDocumentScannerPlugin implements FlutterPlugin, ActivityAware, MethodChannel.MethodCallHandler,
-        PluginRegistry.ActivityResultListener {
 
+public class GoogleMlKitDocumentScannerPlugin implements FlutterPlugin, ActivityAware {
```

Comment:
Okay, i got it. 
This is was what i miss. 
I was implementing ActivityAware in DocumentScanner class